library/src/main/java/com/novoda/downloadmanager/lib/DownloadReadyChecker.java

```diff
@@ -23,13 +23,17 @@
 
     public boolean canDownload(DownloadBatch downloadBatch) {
         if (isDownloadManagerReadyToDownload(downloadBatch)) {
-            Download download = downloadMarshaller.marshall(downloadBatch);
-            return downloadClientReadyChecker.isAllowedToDownload(download);
+            return clientAllowsToDownload(downloadBatch);
         }
 
         return false;
     }
 
+    boolean clientAllowsToDownload(DownloadBatch downloadBatch) {
```

Comment:
Because it does not need to be public :)
Used in the `DownloadTask` class in the same package